doc/barclamp.md

```diff
@@ -1356,3 +1356,154 @@ place) you will have to add additional setup code to
 and/or
 [qa_crowbarsetup.sh](https://github.com/SUSE-Cloud/automation/blob/master/scripts/qa_crobarsetup.sh)
 to take care of your barclamp's needs.
+
+## High Availability (HA) Integration
+
+Using the [barbican barclamp](https://github.com/crowbar/crowbar-openstack/tree/master/chef/cookbooks/barbican)
+as an example. We will try to investigate what changes are required to make a barclamp
+HA compatible.
+
+[This commit](https://github.com/crowbar/crowbar-openstack/pull/512/files) is
+the first place we start looking. It was not entirely correct and needed
+[more commits](https://github.com/crowbar/crowbar-openstack/commits/master/chef/cookbooks/barbican)
+to correct. There are a lot of potential mistakes in getting HA working, maybe 
+whatever you are struggling with right now turned up in the Barbican barclamp as well.
+
+-  [default.rb](https://github.com/crowbar/crowbar-openstack/commits/master/chef/cookbooks/barbican/attributes/default.rb)
+
+   This file defines the defaults for all the values of the vars required by barclamp.
+It allows you to set defaults (or overrides) for any attributes in the node object.
+In this case we will use that capability to set a bunch of HA-related attributes that
+are rarely changed and, hence need not be put int the data bag.
+
+```ruby
+# HA attributes
+default[:barbican][:ha][:enabled] = false
+
+# Ports for actual application to bind to
+default[:barbican][:ha][:ports][:api] = 5621
+
+# pacemaker definitions
+default[:barbican][:ha][:api][:op][:monitor][:interval] = "10s"
+default[:barbican][:worker][:op][:monitor][:interval] = "10s"
+default[:barbican][:keystone_listener][:op][:monitor][:interval] = "10s"
+
+default[:barbican][:worker][:agent] = "systemd:openstack-barbican-worker"
+default[:barbican][:keystone_listener][:agent] = "systemd:openstack-barbican-keystone-listener"
+```
+
+> - Resource Agent: In 'LayMan' terms, resource agent is a script that the
```

Comment:
@sjamgade I appreciate the intention, but this doesn't cause it to be rendered any differently, does it? In any case `>` is meant for block-quoting only, so I don't think it's right to abuse it for something else.